lib/mongoose_push/application.ex

```diff
@@ -32,7 +32,7 @@ defmodule MongoosePush.Application do
     MongoosePush.Telemetry.attach_all()
 
     # Define workers and child supervisors to be supervised
-    children = children()
+    children = children() ++ [MongoosePushWeb.Endpoint]
```

Comment:
Maybe let's rename `children` to `service_children` or something like that in order to indicate that those are not "all possible children" anymore.